### PR TITLE
Add after update wizard

### PR DESCRIPTION
**Pull request by mariogrip**
This adds update wizard that will show after an change to framework has been done. This also adds new pages, one to reinstall/update all apps and one to update/check for firmware update.

REQUIRED PRS:
https://github.com/ubports/system-settings/pull/71
https://github.com/ubports/system-image/pull/5
https://github.com/ubports/ubuntu-touch-meta/pull/23

!don't squash!

### CODE REVIEW

File: plugins/Wizard/System.cpp [hunk 1]
```diff
@@ -46,29 +47,82 @@ QString System::wizardEnabledPath()
...
+    QFileInfo f("/usr/share/click/frameworks/current");
+    return f.canonicalFilePath();
+}
+
+/*
+wizardEnabled and isUpdate logic
+
+if wizard-has-run does NOT exists == is new install
+if wizard-has-run does exists but does NOT match current framework == is update
+if wizard-has-run does exists but does match current framework == show no wizard
```
Comment (NeoTheThird) [inline plugins/Wizard/System.cpp]: "does exist" and "does not exist"

File: plugins/Wizard/System.cpp [hunk 2]
```diff
@@ -46,29 +47,82 @@ QString System::wizardEnabledPath()
...
+*/
+
+bool System::wizardPathExists() {
+  return QFile::exists(wizardEnabledPath());
+}
+
 bool System::wizardEnabled() const
 {
-    return !QFile::exists(wizardEnabledPath());
+  if (!wizardPathExists())
```
Comment (NeoTheThird) [inline plugins/Wizard/System.cpp]: indentation width should be four spaces for c++

File: qml/Wizard/Pages.qml [hunk 3]
```diff
@@ -105,6 +106,39 @@ StyledItem {
...
+            to: 1
+            duration: 200
+        }
+
+        onVisibleChanged: {
+            if (visible) {
+                opacity = 0;
+                fadeInAnimation.start();
+            } else {
+                // impatientLoadingLabel.text = ""
```
Comment (NeoTheThird) [inline qml/Wizard/Pages.qml]: ?

**Review by NeoTheThird — CHANGES_REQUESTED** [review]

File: qml/Wizard/Pages.qml (hunk 3 above)
Comment (mariogrip) [reply qml/Wizard/Pages.qml]: summoning @UniversalSuperBox since he wrote that part ^

**Comment by mariogrip** [comment]
@NeoTheThird?

**Review by NeoTheThird — APPROVED** [review]

**Pull request merged**